stages/org.osbuild.erofs

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/python3
+"""
+Create a file containing an errofs filesystem named `filename`.
```

Comment:
`errofs` or `erofs`? I think it's a typo. It might also benefit from a link to information about the filesystem, I'd never heard of it before.

Reply:
Thanks, nice catch! Indeed a typo, I fixed via a force push and also added a link to the wikipedia page, it's a very cool new one and IIRC it is used by composefs.